### PR TITLE
Package delimcc_of_fxhandler.0.1

### PR DESCRIPTION
**Pull request by kayceesrk**
### `delimcc_of_fxhandler.0.1`
Delimited control primitives on top of effect handlers
Delimited control primitives on top of effect handlers

---
* Homepage: https://github.com/kayceesrk/delimcc_of_fxhandler
* Source repo: git+https://github.com/kayceesrk/delimcc_of_fixhandler.git
* Bug tracker: https://github.com/kayceesrk/delimcc_of_fxhandler/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

File: packages/delimcc_of_fxhandler/delimcc_of_fxhandler.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+tags: ["effect handlers" "delimited control"]
+homepage: "https://github.com/kayceesrk/delimcc_of_fxhandler"
+bug-reports: "https://github.com/kayceesrk/delimcc_of_fxhandler/issues"
+depends: [
+  "ocaml" {>= "5.0.0"}
+  "multicont"
+  "dune" {>= "3.7"}
+  "odoc" {with-doc}
+]
+build: [
```
Comment (avsm) [inline packages/delimcc_of_fxhandler/delimcc_of_fxhandler.0.1/opam:17]:
```suggestion
conflicts: ["ocaml-option-bytecode-only"]
build: [
```

Is this intended to work with bytecode-only backends?  This option above should avoid this being selected on those (arm32, etc)

Comment (kayceesrk) [reply packages/delimcc_of_fxhandler/delimcc_of_fxhandler.0.1/opam:18]: Thanks for the suggestion.

**Comment by mseri** [comment]
Something strange is going on:
```

#=== ERROR while compiling delimcc_of_fxhandler.0.1 ===========================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/kayceesrk/delimcc_of_fxhandler/archive/refs/tags/0.1.tar.gz)
# path                 ~/.opam/5.0/.opam-switch/build/delimcc_of_fxhandler.0.1
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p delimcc_of_fxhandler -j 255 @install
# exit-code            1
# env-file             ~/.opam/log/delimcc_of_fxhandler-7-f6ec2e.env
# output-file          ~/.opam/log/delimcc_of_fxhandler-7-f6ec2e.out
### output ###
# File "/home/opam/.opam/5.0/lib/multicont/dune-package", line 1, characters 0-846:
# 1 | (lang dune 3.0)\n(name multicont)\n(version 1.0.0~rc.3)\n(sections (lib .) (stublibs ../../stublibs) (doc ../../doc/multicont))\n(files\n (lib\n  (multicont.mli\n   multicont.a\n   multicont.cma\n   multicont.cmi\n   multicont.cmti\n   multicont.cmx\n   multicont.cmxa\n   libmulticont.a\n   libmulticontopt.a\n   META\n   dune-package))\n (stublibs\n  (dllmulticont.so\n   dllmulticontopt.so))\n (doc\n  (LICENSE\n   README.md)))\n(library\n (name multicont)\n (synopsis "Multi-shot continuations in OCaml")\n (kind normal)\n (archives (byte multicont.cma) (native multicont.cmxa))\n (foreign_archives libmulticont.a libmulticontopt.a)\n (native_archives multicont.a)\n (main_module_name Multicont)\n (modes byte native)\n (modules\n  (singleton\n    (name Multicont)\n    (obj_name multicont)\n    (visibility public)\n    (impl)\n    (intf))))
#     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Invalid first line, expected: (lang <lang> <version>)
# -> required by library "delimcc_of_fxhandler" in _build/default/lib
# -> required by _build/default/META.delimcc_of_fxhandler
# -> required by _build/install/default/lib/delimcc_of_fxhandler/META
# -> required by _build/default/delimcc_of_fxhandler.install
# -> required by alias install
```

**Comment by kayceesrk** [comment]
The errors are in the `multicont` library, one of the dependencies of the current library. What's the expectation here regarding dependencies failing?

**Comment by mseri** [comment]
Yes, I don’t consider it a blocker for this PR.

The PR is still open because I needed to check if my fix for 5e lower bounds was enough. I am rerunning those two tests now that it is merged and if they pass, I will merge

**Comment by mseri** [comment]
All good, thanks!

**Comment by avsm** [comment]
@dhil, the ocaml-multicont library seems to be playing gymnastics with dune :-)
https://github.com/dhil/ocaml-multicont/blob/master/Makefile#L98

**Comment by dhil** [comment]
Hopefully fixed by #23988.

**Pull request merged**